src/Bluejay.asm

```diff
@@ -446,366 +444,362 @@ $include (Modules\Settings.asm)
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 
-
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 ;
 ; Main program entry point
 ;
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 pgm_start:
-	mov	Flash_Key_1, #0			; Initialize flash keys to invalid values
-	mov	Flash_Key_2, #0
-	mov	WDTCN, #0DEh				; Disable watchdog (WDT)
-	mov	WDTCN, #0ADh
-	mov	SP, #Stack				; Initialize stack (16 bytes of indirect RAM)
-	IF MCU_TYPE < 2
-		orl	VDM0CN, #080h			; Enable the VDD monitor
-	ENDIF
-	mov	RSTSRC, #06h				; Set missing clock and VDD monitor as a reset source if not 1S capable
-	mov	CLKSEL, #00h				; Set clock divider to 1 (Oscillator 0 at 24MHz)
-	call	switch_power_off
-	; Ports initialization
-	mov	P0, #P0_INIT
-	mov	P0MDIN, #P0_DIGITAL
-	mov	P0MDOUT, #P0_PUSHPULL
-	mov	P0, #P0_INIT
-	mov	P0SKIP, #P0_SKIP
-	mov	P1, #P1_INIT
-	mov	P1MDIN, #P1_DIGITAL
-	mov	P1MDOUT, #P1_PUSHPULL
-	mov	P1, #P1_INIT
-	mov	P1SKIP, #P1_SKIP
-	mov	P2MDOUT, #P2_PUSHPULL
+mov                           Flash_Key_1, #0     ; Initialize flash keys to invalid values
```

Comment:
I am used to read the code with 4 spaces indentation in instructions and 0 in tags and preprocessor commands. Please, could it be possible to change to that style?

Reply:
Sure, can add this. So 4 spaces intendation should be the default, also for IF's - right?

Reply:
Yes